pkg/apis/ceph.rook.io/v1/network_test.go

```diff
@@ -43,11 +43,32 @@ func TestNetworkCephSpecLegacy(t *testing.T) {
 }
 
 func TestNetworkCephIsHostLegacy(t *testing.T) {
-	net := NetworkSpec{HostNetwork: true}
+	net := NetworkSpec{ HostNetwork: true}
```

Comment:
@BlaineEXE  wrote:
> I'm not sure how this space got added.

I don't know either: At least I didn't intentionally modify that line.

My usual suspect for such unexpected changes is the editor. I don't like the editor to do a lot without me entering the changes myself. At least I usually try to undo formatting changes by the editor as soon as I notice them. but that is usually of course not very successful  😉 

I prefer to run `gofmt` manually. I just did and  after committing  the result now the extra space is gone.

Reply:
I guess this check must not be inour CI, but Go code must be formatted with `gofmt` and should not be changed afterwards to alter its formatting. 

I think we haven't noticed it before because most people use VSCode and the Golang plugin that does this automatically. I'm surprised that our linter doesn't check for it as well.